cmd/up/forwards_test.go

```diff
@@ -0,0 +1,112 @@
+package up
+
+import (
+	"context"
+	"testing"
+
+	"github.com/okteto/okteto/pkg/model"
+	"github.com/okteto/okteto/pkg/ssh"
+)
+
+func TestGlobalForwarderStartsWhenRequired(t *testing.T) {
+	var tests = []struct {
+		name           string
+		manifestInfo   []model.Forward
+		expectedAnswer bool
+	}{
+		{
+			name: "is needed global forwarding",
+			manifestInfo: []model.Forward{
+				{
+					Local:  8080,
+					Remote: 8080,
+				},
+			},
+			expectedAnswer: true,
+		},
+		{
+			name:           "not needed global forwarding",
+			manifestInfo:   []model.Forward{},
+			expectedAnswer: false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			answer := isNeededGlobalForwarder(tt.manifestInfo)
+			if answer != tt.expectedAnswer {
```

Comment:
I think we agreed on start using assertify library for testing. Tests looks much cleaner :)